docs/config.rst

```diff
@@ -243,6 +243,23 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_CONTAINER
+   :type: bool
+   :value: 0
+
+   Indicate if the shell is running in a container environment (e.g.
+   Docker, Podman, LXC, Singularity)
+
+   Note: Containers may inherit some or even no variables from their parent
```

Comment:
Use a `note` block:
```suggestion
   .. note::
      Containers may inherit some or even no variables from their parent
```
Make sure to realign the whole block, and keep the wrapping at 80 chars.